File: javascript/1603.js
/**
 * @param {number} big
 * @param {number} medium
 * @param {number} small
 */
var ParkingSystem = function (big, medium, small) {
    this.parking = [big, medium, small]
};

/**
 * @param {number} carType
 * @return {boolean}
 */
ParkingSystem.prototype.addCar = function (carType) {
    this.parking[carType - 1] -= 1
    return this.parking[carType - 1] >=  0
};

File: python/1603.py
class ParkingSystem:

    def __init__(self, big: int, medium: int, small: int):
        self.parking = [big, medium, small]

    def addCar(self, carType: int) -> bool:
        self.parking[carType - 1] -= 1
        return self.parking[carType - 1] >= 0
